src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -50,15 +49,14 @@ protected void prepareTestData() {
                 .builder("New Session ##", course.getId(), "teammates.test1@gmail.tmt")
                 .withStartTime(TimeHelper.parseInstant("2035-04-01 9:59 PM +0000"))
                 .withEndTime(TimeHelper.parseInstant("2035-04-30 8:00 PM +0000"))
-                .withCreatedTime(Const.TIME_REPRESENTS_NEVER)
+                .withCreatedTime(Const.TIME_REPRESENTS_NOW)
```

Comment:
Was this needed to fix the test? If so, would an arbitrary timestamp be better than the special timestamp (I don't think the special one is possible in real usage)?